server/events/project_command_runner_test.go

```diff
@@ -451,8 +458,6 @@ func TestDefaultProjectCommandRunner_Apply(t *testing.T) {
 				switch step {
 				case "approved":
 					mockApproved.VerifyWasCalledOnce().PullIsApproved(ctx.BaseRepo, ctx.Pull)
-				case "mergeable":
-					mockMergeable.VerifyWasCalledOnce().PullIsMergeable(ctx.BaseRepo, ctx.Pull)
 				case "init":
 					mockInit.VerifyWasCalledOnce().Run(ctx, nil, repoDir)
 				case "plan":
```

Comment:
Can you add a test case that tests your new code?

Reply:
How would that work? Sorry, I'm not sure how this mocking framework works, but it seems like you're testing if a function is being called; whereas in the case of testing Mergeability, its just checking the value of `ctx.PullMergeable`

Reply:
@lkysow Added a test case to test the failure path. not sure if that's what you were asking for; lemme know

Reply:
That's perfect